<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html lang="en-US" xml:lang="en-US" xmlns="http://www.w3.org/1999/xhtml">

<head>
<link rel="stylesheet" type="text/css" href="http://gtpb.igc.gulbenkian.pt/bicourses/GTPB2015style.css">
</link>
<title>
GTPB: PDA16 - Proteomics Data Analysis  - Home
</title>
</head>

<body>

<!-- GTPB header -->

<table width="910" align="center" >
<tr><td>
<img src="../images/GTPB2015logo.png" width="890" align="center" >
</td></tr>
</table>


<table width="910" align="center" >
<tr><td>
<br />
<font size="6" color="#006400"><b>PDA16</b></font>
<br />
<h1>Proteomics Data Analysis</h1>


<b><a href="./../posters/PDA16.pdf">Downloadable poster in PDF</a><b><br>

</td></tr>
</table>

<table width="910" align="center">
<font size="2">
<tr><td>
<p><b>&nbsp;&nbsp;&nbsp;IMPORTANT DATES</b> for this Course
<br />&nbsp;&nbsp;&nbsp;Deadline for applications: <b>April 2nd 2016</b> (New)
<br />&nbsp;&nbsp;&nbsp;Course date: <b>April 11th -  April 15th 2016</b>
</p>
<p><b>Candidates with adequate profile will be accepted in the next 72 hours after the application until we reach 20 participants.
</td></tr>
</font>
</table>


<table width="910" align="center">
<tr><td>
<h2>Instructors:</h2>
</td></tr>
<tr><td >
<img border="0" style="margin:4px" src="../images/Lennart_Martens.jpg" width="91"  align="left"></img>
<font size="2"><p align="justify">
<b>Lennart Martens</b>  Lennart Martens is professor of systems biology at Ghent University, and  group leader of the Computational 
Omics and Systems Biology (CompOmics) group at VIB, both in Ghent, Belgium. He has been working in proteomics bioinformatics since 
his Master's degree, which focused on the computational interpretation of peptide mass spectra. He then worked as a software developer 
and framework architect for a software company for a few years, before returning to Ghent University to pursue a Ph.D. focused on 
proteomics and proteomics informatics. During this time, he worked on the development of high-throughput peptide centric proteomics 
techniques and on bioinformatics tools to support these new approaches. In 2003 he started the PRIDE proteomics database at the EBI 
as a Marie Curie fellow of the European Commission. After obtaining his Ph.D., he rejoined the PRIDE group at EBI, which he coordinated 
for several years before moving back to Ghent University to take up his current position. Prof. Martens served as the chair of the ABRF 
iPRG in 2011, sits on the Board of DELSA (http://www.delsaglobal.org), the Royal Flemish Society for Chemistry - Proteomics, and the 
Belgian Proteomics Association. He serves as Academic Editor for PLoS ONE and Open Proteomics and holds Editorial Board positions at 
PROTEOMICS, BBA Proteins and Proteomics and Molecular BioSystems. An author on more than 135 peer-reviewed papers and has co-written 
two popular Wiley textbooks: Computational Methods for Mass Spectrometry Proteomics (ISBN: 978-0-470-51297-5), and Computational and 
Statistical Methods for Protein Quantification by Mass Spectrometry (ISBN: 978-1-119-96400-1).
</p>

<b>Affiliation:</b>  Ghent University and VIB, Ghent, BE<br>
</td></tr>

<tr><td >
<img border="0" style="margin:4px" src="../images/Harald_Barsnes.jpg" width="91"  align="left"></img>
<font size="2"><p align="justify">
<b>Harald Barsnes</b>  currently works as a researcher at the Department of Clinical Science at the 
University of Bergen in Norway. He has a PhD from the Department of Informatics at the University of 
Bergen in 2010, with a focus on bioinformatics and proteomics. Since then he has worked as a first a 
senior engineer and then a postdoc at the Proteomics Unit at the Department of Biomedicine in Bergen. 
The main focus throughout his career has been the development of user-friendly open source bioinformatics 
tools that enable and empower researchers to analyze their own proteomics data. This has resulted in 
numerous publications and a long list of freely available software, including PRIDE Converter, SearchGUI 
and PeptideShaker. Barsnes has also been a guest editor in PROTEOMICS and has co-authored a Wiley text 
book called Computational and Statistical Methods for Protein Quantification by Mass Spectrometry (ISBN: 978-1-1199-6400-1). 
In 2015 he received the prestigious Meltzer Award for Excellent Young Researchers at the University of Bergen.
</p>

<b>Affiliation:</b> Department of Clinical Science, University of Bergen, NO<br>
</td></tr>

<tr><td >
<img border="0" style="margin:4px" src="../images/Marc_Vaudel.jpg" width="91"  align="left"></img>
<font size="2"><p align="justify">
<b>Marc Vaudel</b> currently works as a postdoc at the Department of Biomedicine at the University of Bergen, 
Norway. Marc obtained an engineer degree in France at the &Eacute;cole Centrale de Lyon, followed by a PhD at the 
biomedical department of the Ghent University, Belgium, in cooperation with the Leibniz - Institut f&uuml;r Analytische 
Wissenschaften - ISAS - e.V. in Dortmund, Germany, under the joint supervision by Prof. Dr. Lennart Martens (Gent, Belgium) 
and Prof. Dr. Albert Sickmann (Dortmund, Germany). His main focus is the investigation of biomedical samples using proteomics. 
Marc is also an associated member of the Computational Omics and Systems Biology (CompOmics) group at VIB in Ghent, headed by 
Prof. Dr. Lennart Martens, of the Bergen stem cell consortium headed by Prof. Dr. Helge R&aelig;der, and member of the educational 
committee of the European Proteomics Association (EuPA).
</p>

<b>Affiliation:</b> Department of Biomedicine, University of Bergen, NO<br>
</td></tr>


</td></tr>

<tr><td>
<h1>Course description</h1>
Mass spectrometry based proteomic experiments generate ever larger datasets and, as a consequence, complex data interpretation 
challenges. In this course, the concepts and methods required to tackle these challenges will be introduced, covering both protein 
identification and quantification. The course will rely exclusively on free and user friendly software, all of which can be directly 
applied in your lab upon your return from the course.<br><br>
An introduction to the available online resources and repositories will also be given. Here you will see how to link the results from 
proteomic experiments with external data to conduct pathway, gene ontology and interaction analyses.<br>
In the course, you will also learn how to submit data to the ProteomeXchange online repositories, and how to browse and reprocess publicly 
available data from these repositories.<br><br>
The course will provide a solid basis for beginners, but also new perspectives to those already familiar with standard data interpretation procedures in proteomics.
 <br>


<h1>Course Pre-requisites</h1>
The participants should have a basic knowledge about mass spectrometry
based proteomics. Experience in analyzing proteomics data is an
advantage, but not mandatory. The course does not require advanced
computer skills.
</td></tr>
</table>

<table width="910" align="center" >
<tr><td>
<br /><a href="./Application.html"><b>Application</b></a>
</td></tr>

<tr><td>
<br /><a href="./timetable.html"><b>Detailed Program </b></a> 
</td></tr>
</table>


<table width="910" align="center">
<tr><td>

<p>
Instituto Gulbenkian de Ci&ecirc;ncia,</p>
<p>
Apartado 14, 2781-901 Oeiras, Portugal</p>
<p><a href="http://gtpb.igc.gulbenkian.pt/bicourses">GTPB Homepage</a></p>
<p><a href="http://www.igc.gulbenkian.pt">IGC Homepage</a></p>
<p>Last updated:&nbsp;&nbsp;<i><b>Feb 26th 2016</b></i></p>
</td></tr>
</table>
</body>

</html>
